packer/configure-windows-vhd.ps1

```diff
@@ -67,6 +67,7 @@ function Get-FilesToCacheOnVHD
         "c:\akse-cache\win-k8s\" = @(
             "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.6-1int.zip",
             "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.7-1int.zip",
+            "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.8-1int.zip",

```

Comment:
Looks like we already had the 1.15.5 artifact config here?

Reply:
Actually forgot to add it, thanks